module/scripts/item/updateItem.js

```diff
@@ -0,0 +1,14 @@
+import { emitForGM } from '../socket/socketMessage.js';
+
+export function updateItem(item, data) {
+    if(item.canUserModify(game.user, 'update')) {
+        item.update(data)
+    } else {
+        emitForGM('updateItemByUuid', [item.uuid, data])
+    }
+}
+
+export function updateItemByUuid(itemId, data) {
+    const item = fromUuidSync(itemId)
+    item?.update(data)
```

Comment:
I didnt add generic update calls to prevent abuse and user errors. Allowing anyone to just update an item with any data is prone to produce problems in the long term